source/WebApp/components/internal/codemirror/mode-asm.ts

```diff
@@ -8,11 +8,11 @@ CodeMirror.defineMode('asm', () => {
         builtin: instructionsRegex
     };
     const specifiers = [ 'byte', 'dword', 'ptr', 'qword', 'short', 'tbyte', 'word' ];
-    const registers = [ 'ah', 'al', 'ax', 'bh', 'bl', 'bp', 'bpl', 'bx', 'ch', 'cl', 'cx', 'dh', 'di', 'dil',
-        'dl', 'dx', 'eax', 'ebp', 'ebx', 'ecx', 'edi', 'edx', 'esi', 'esp', 'rax', 'rbp', 'rbx', 'rcx', 'rdi',
-        'rdx', 'rsi', 'rsp', 'r8', 'r9', 'r10', 'r11', 'r12', 'r13', 'r14', 'r15', 'r8b', 'r9b', 'r10b', 'r11b',
-        'r13b', 'r14b', 'r15b', 'r8d', 'r9d', 'r10d', 'r11d', 'r12d', 'r13d', 'r14d', 'r15d', 'r8w', 'r9w',
-        'r10w', 'r11w', 'r12w', 'r13w', 'r14w', 'r15w' ];
+    const registers = [ 'rsp', 'rsi', 'rdx', 'rdi', 'rcx', 'rbx', 'rbp', 'rax', 'r15w', 'r15d', 'r15b', 'r15',

```

Comment:
Since it's pre-sorted, I think it would be good to add a comment explaining that it is pre-sorted, otherwise when changing something ppl might not realize they mess up the sort.

Reply:
Comment added.